client/src/components/employer-form/index.js

```diff
@@ -10,32 +10,48 @@ import { scrollUp } from '../../utils';
 
 import { Card, Button } from '../generic';
 import { Summary } from './Summary';
-import { BasicInfo} from './BasicInfo';
+import { OperatorInfo } from './OperatorInfo';
+import { SiteInfo} from './SiteInfo';
```

Comment:
Okay I see this missing space, it's fixed locally and will push it up with any other changes from the PR lol